src/components/OSCALControlProse.js

```diff
@@ -384,6 +385,9 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
   // https://github.com/EasyDynamics/oscal-react-library/issues/499
   const statementByComponentDescription =
     statementByComponent?.description || null;
+  const statementByComponentDescriptionMarkdown = (
```

Comment:
To stay consistent with discussions from #548 on markdown vs markup, can we change this to:
```suggestion
  const statementByComponentDescriptionMarkup = (
```

Reply:
That makes sense! My most recent commit has this change